scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
+    uri = URL(DOMAIN+RANDOM)
+    response = Net::HTTP.get_response(uri)
+    matchdata = REGEXP.match(response['Location'])
```

Comment:
`match_data` is indeed better, even my Python skills lacked here.

Since it's a the regular expression is a constant I think the All Caps approach is better as referred [here](https://github.com/bbatsov/ruby-style-guide#screaming-snake-case).